apps/main-app/src/components/paralogy/paralogyFilteredTable.js

```diff
@@ -0,0 +1,235 @@
+    data.results[0].predictionMethodsNotMatched
+  ).sort(compareAlphabeticalCaseInsensitive);
+
+  const labelStyle = {
+    margin: '0em 1em 0em 0',
+    lineHeight: '2em',
+  };
+  const inputStyle = {
+    margin: '0 0.5em'
+  };
+
+  const buttonStyle = {
+    marginTop: '0.5em',
+    marginRight: '0.5em',
+    minWidth: '8em',
+  };
+
+  const docTextStyle = {
+    fontStyle: 'italic',
+    opacity: 0.7,
+    fontSize: '0.9em',
+  };
+
+  return (
+    <div>
+      <ControlsContainer>
+        <span className='pull-right'>
+          <HelpPopup id='orthology-controls-help'>
+            <OrthologyFilterHelp />
+          </HelpPopup>
+        </span>
+        <StringencySelection
+          level={stringencyLevel}
+          onChange={setStringencyLevel}
+        />
+        <Collapse isOpen={showFilterPanel}>
+          <div>
+            <span style={docTextStyle}>Additional filters to further constrain the results:</span>
+            <div style={{display: 'flex', flexDirection: 'column', marginLeft: '-0.5em'}}>
+              <label style={labelStyle}>
+                <input
+                  checked={filterBest}
+                  onChange={(event) => setFilterBest(event.target.checked)}
+                  style={inputStyle}
+                  type="checkbox"
+                />
+                  Best Score Only
+              </label>
+              <label style={labelStyle}>
+                <input
+                  checked={filterReverseBest}
+                  onChange={(event) => setFilterReverseBest(event.target.checked)}
+                  style={inputStyle}
+                  type="checkbox"
+                />
+                  Best Reverse Score Only
+              </label>
+            </div>
+            <label style={labelStyle}>
+                Count:
+              <select
+                onChange={(event) => setFilterScoreGreaterThan(+event.target.value)}
+                style={inputStyle}
+                value={filterScoreGreaterThan}
+              >
+                <option value={0}>&gt; 0</option>
+                {
+                  all_methods.map((method, index) => {
+                    const scoreGreaterThanValue = index + 1;
+                    return <option key={scoreGreaterThanValue} value={scoreGreaterThanValue}>&gt; {scoreGreaterThanValue}</option>;
+                  })
+                }
+              </select>
+            </label>
+            <label style={labelStyle}>
+                Species:
+              <select
+                onChange={(event) => setFilterSpecies(event.target.value === 'all' ? null : event.target.value)}
+                style={inputStyle}
+                value={filterSpecies || 'all'}
+              >
+                <option value="all">All</option>
+                {
+                  data.results.reduce((all_species, dat) => {
+                    if (all_species.indexOf(getOrthologSpeciesName(dat)) === -1) {
```

Comment:
Paralogy does not have different species. I think this is superfluous...

Reply:
Removed!